.github/workflows/dev-container.yml

```
@@ -26,13 +28,51 @@ jobs:
-          # TODO: @alexanderilyin: Allure
-          # TODO: @alexanderilyin: Code Coverage
-          runCmd: .github/scripts/dev-container.sh
+          # runCmd: .github/scripts/dev-container.sh
+  install:
+    # TODO: @alexanderilyin: cache "poetry install"
+    name: Install Python dependencies
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout (GitHub)
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)